File: beginner/python/2547.py
while True:
    try:    
        n, nn, nm = list(map(int, input().split()))

        cont = 0
        for i in range(n):
            x = int(input())
            if (x >= nn and x <= nm):
                cont+=1

        print(cont)
    except EOFError:
        break

File: beginner/javascript/2547.js
var fs = require("fs");
var input = fs.readFileSync("stdin", "utf8");
var lines = input.trim().split("\n");

var i = 0;

while (i < lines.length) {
    const [n, nn, nm] = lines[i].split(" ").map(Number);
    i++;

    let cont = 0;

    for (let j = 0; j < n; j++) {
        const x = parseInt(lines[i]);
        i++;
        if (x >= nn && x <= nm) {
            cont++;
        }
    }

    console.log(cont);
}
